subsys/net/ip/Kconfig

```diff
@@ -494,6 +495,16 @@ config NET_TCP_RECV_QUEUE_TIMEOUT
 	  SEQ 2. But if we receive SEQs 5,4,3,7 then the SEQ 7 is discarded
 	  because the list would not be sequential as number 6 is be missing.
 
+config NET_TCP_PKT_ALLOC_TIMEOUT
+	int "How long to wait for a TCP packet allocation (in ms)"
+	depends on NET_TCP
+	default 100
```

Comment:
Should we enforce some sane limits here, now it is possible to put any int value.
Perhaps something like `range 10 1000` or similar. The lower limit should not be too small as that would cause allways a timeout.

Reply:
Thank you, I've put the range